src/panel/NoResultPanel.jsx

```diff
@@ -14,13 +15,12 @@ const NoResultPanel = ({ resetInput }) => {
   return (
     <Panel close={close}>
       <div style={{ padding: '16px 16px 32px 16px' }}>
-        <p className="u-mb-xs u-text--smallTitle">{_('No results found.')}</p>
-        <p className="u-text--subtitle u-mb-l">
-          {_('Check the spelling of your search or add more details, such as city or country.')}
+        <NoResultMessage />
+        <p className="u-center">
+          <a onClick={tryNewSearch} href="#">
+            {_('Try a new search query')}
+          </a>
```

Comment:
The margin above the link has been lost here
![image](https://user-images.githubusercontent.com/4726554/119640838-4b329c80-be19-11eb-853f-ab409f8cc5d0.png)

Reply:
I adjusted the spaces around the message, and above the link according to the designs